### DISCUSSION TITLE
Document states first android app must manually be added to google play, but doesn't say how

### DISCUSSION THREAD

**Discussion by snowch** [codemagic-ci-cd/codemagic-docs#2387] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

- Flutter (Dart)

### Steps to reproduce

1. Trying to publish to Google Play as per document: https://docs.codemagic.io/flutter-publishing/publishing-to-google-play/
2. Documents states "The very first version of the app must be added to Google Play manually", however, it doesn't say how to do this. 

### Expected results

I would expect documentation to have either the steps for manually uploading the first version of an app or links to the steps.

### Actual results

No documentation on steps.

**Comment by dtrdic** [comment, marked as answer]
Hello @snowch, 
1. you will first need to create a new app on [Google play console](https://play.google.com/console/u/0/developers/8478664491092592353/create-new-app)![Screenshot 2024-02-21 at 11 28 14](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/32ffe78c-d2cf-4cef-98f2-7a6763ec5774)

2. After you fill in all the required information, look for the `Create new release` button, and upload the binary that was produced with Codemagic

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/17865b1f-4819-41d3-8cbd-c923e58e492a)

**Comment by icarusdust** [comment]
Thanks for the suggestion @snowch 

We will consider adding this info to the documentation.
